agenta-cli/agenta/sdk/types.py

```diff
@@ -127,6 +127,56 @@ def __modify_schema__(cls, field_schema: dict[str, Any]):
         )
 
 
+class GroupedMultipleChoiceParam(str):
+    def __new__(cls, default: str = None, options: Dict[str, List[str]] = None):
+        if options is None:
+            options = {}
+
+        # Check if default is in the options
+        if default and not any(default in choices for choices in options.values()):
+            if not options:
+                print(
```

Comment:
That should not be a warning, no? We should not allow a None value to the options, no? So basically an assert